### PR TITLE
Fixed gtrak10 video.

### PR DESCRIPTION
**Pull request by couriersud**
Please ensure that gtrak10.cfg is deleted - may override defaults.

### CODE REVIEW

**Comment by felipesanches** [comment]
This is looking great! The spurious pixels on the top (which are game data, per https://github.com/mamedev/mame/pull/10013#issuecomment-1172266961) are not show anymore, which I expect to be correct behavior, but there's still a small amount of such pixels in the bottom. I guess those where also not visible on the original cabinet (but I have no evidence to back this as I have no access to the real hardware to test).
 
Screenshot from 2022-07-01 19-19-44


The finish-line (a vertical dotted one on the lower left portion of the track) seems to have disappeared.
Here that line can be seen prior to this PR, while using the hacks mentioned at https://github.com/mamedev/mame/pull/10013#issuecomment-1171958523 and on driver comments:
Screenshot from 2022-07-01 04-06-50

**Comment by goldnchild** [comment]
I tried to make the changes by hand and it wouldn't work, then after awhile I realized that I typed 1.0 for the 0.1 in m_video->set_vsync_threshold(0.1);

Looks good, wonder why the car image seems to have a "temporal" smoothing effect, is that a "deinterlacing" effect?

Also I'm going to have to figure out why the car appears offscreen. 8-)

**Comment by couriersud** [comment]
> t's interesting that there's a slight interlace jitter when it starts up for a few frames, then it enables temporal smoothing.
> (Is there any way to turn off smoothing so you can see the jitter?)

Use mame without hsl.

The real performance killers on this game are C9, R1,R2 which are used to genereate a very short pulse to reset the logic around L1. I have other netlist changes in the pipeline and will provide a logic replacement once these are in master.

There are pullup resistor which are not in the schematic connected to the switches. This should not be necessary - see e.g. breakout.

File: src/mame/atari/atarittl.cpp [hunk 1]
```diff
@@ -293,26 +284,27 @@ void gtrak10_state::gtrak10(machine_config &config)
...
+	   Horiz Sync        =  32
+	   Horiz Back Porch  = ?
 
 	   Vert Total       = 521
-	   Vert Front Porch =   0
-	   Vert Sync        =   8
-	   Vert Back Porch  =   0
+	   Vert Front Porch =   ?
+	   Vert Sync        =   4
+	   Vert Back Porch  =   ?
```
Comment (cuavas) [inline src/mame/atari/atarittl.cpp:294]: Are you sure about this?  Most monitors require exactly three lines of vertical sync and three lines of vertical front porch.  At any rate, zero for front porch and back porch is impossible.

Comment (couriersud) [reply src/mame/atari/atarittl.cpp:294]: Yes, the COMP_SYNC signal shows 4 lines of vertical sync for field 1. For the second field it is 0.5 + 3 + 0.5 = 4 as well starting in the middle of the screen. There is no vertical front porch generated by the logic, vreset directly triggers sync.
The porches are only used to define the visible area. The physical monitor characteristics are unknown, that's why I used `?`.

Comment (cuavas) [reply src/mame/atari/atarittl.cpp:294]: The front porch is also required for equalising pulses in interlaced modes when using composite sync. It isn't purely to define the visible area.

Comment (goldnchild) [reply src/mame/atari/atarittl.cpp:294]: The GT10 manual says that it uses a Motorola XM700 and gives a schematic.  The Lemans manual has a schematic from an XM701.  I was able to find an XM701 service manual at arcade museum by googling "xm701 service manual".

Comment (couriersud) [reply src/mame/atari/atarittl.cpp:294]: The video parameters uses porches:
```
	m_video->set_horz_params(GTRAK10_HTOTAL  - 96, GTRAK10_HTOTAL - 64, GTRAK10_HTOTAL - 32, GTRAK10_HTOTAL);
	m_video->set_vert_params( GTRAK10_VTOTAL - 32, GTRAK10_VTOTAL -  8, GTRAK10_VTOTAL - 4,  GTRAK10_VTOTAL);
```
@goldnchild : That's great news. I am sure with the monitor manual you will get the exact porch values sorted.

**Comment by goldnchild** [comment]
Couriersud: No jumping here

So you don't have any horizontal jumping?  Weird, because on my system with Ubuntu if I don't have the hack it jumps all over the place.

**Pull request merged**